homeassistant/components/analytics_insights/coordinator.py

```diff
@@ -0,0 +1,53 @@
+"""DataUpdateCoordinator for the Homeassistant Analytics integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+
+from python_homeassistant_analytics import (
+    HomeassistantAnalyticsClient,
+    HomeassistantAnalyticsConnectionError,
+    HomeassistantAnalyticsNotModifiedError,
+)
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import CONF_TRACKED_INTEGRATIONS, DOMAIN, LOGGER
+
+
+class HomeassistantAnalyticsDataUpdateCoordinator(
+    DataUpdateCoordinator[dict[str, int]]
+):
+    """A Homeassistant Analytics Data Update Coordinator."""
+
+    config_entry: ConfigEntry
+
+    def __init__(
+        self, hass: HomeAssistant, client: HomeassistantAnalyticsClient
+    ) -> None:
+        """Initialize the Homeassistant Analytics data coordinator."""
+        super().__init__(
+            hass,
+            LOGGER,
+            name=DOMAIN,
+            update_interval=timedelta(hours=1),
```

Comment:
Why did you reduce it from 12h?

Reply:
Since with etags we now have a low load. I lowered it to keep the stats more consistent. So what if you restart at 1:00 and the stats update at 1:30. That means the change won't be tracked until 13:00. This way I keep the variance under an hour instead of 12 hours. I can look into using the last updated property in the json (i know we get that from data.json, not sure about current_data.json)